src/components/Ledger/LedgerView/LedgerGroups.js

```diff
@@ -19,15 +18,15 @@ import {
 import { CHUNK_LIMIT } from '../../../common/const';
 import ConnectionListing from '../../ConnectionListing';
 
-const LedgerGroups = ({ history, funds, currency, mutator, resources }) => {
+const LedgerGroups = ({ history, funds, currency, mutator }) => {
   const fundIds = funds.map(fund => `fundId="${fund.id}"`);
   const chunkedFundIds = chunk(fundIds, CHUNK_LIMIT).map(arr => arr.join(' or '));
+  const [groups, setGroups] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
you've set this to `true` from the start, however, I wonder if there a case when it stays `true` ? I see conditions below and it looks like it's could be a real case

Reply:
I changed the initial state for `isLoading` to `false`. You're right, if there are no funds user will see infinitive loading.